src/coreComponents/schema/schema.xsd

```diff
@@ -2008,7 +2008,7 @@ the relative residual norm satisfies:
 			<xsd:element name="NonlinearSolverParameters" type="NonlinearSolverParametersType" maxOccurs="1" />
 		</xsd:choice>
 		<!--cflFactor => Factor to apply to the `CFL condition <http://en.wikipedia.org/wiki/Courant-Friedrichs-Lewy_condition>`_ when calculating the maximum allowable time step. Values should be in the interval (0,1] -->
-		<xsd:attribute name="cflFactor" type="real64" default="0.5" />
+		<xsd:attribute name="cflFactor" type="real64" default="-1" />
```

Comment:
should it be committed?

Reply:
No, you're right, just revert it in a later commit (hopefully)